src/components/Superfest/NFTClaimingBox/NFTClaimingBox.style.ts

```diff
@@ -0,0 +1,99 @@
+import type { BoxProps, Breakpoint } from '@mui/material';
+import { Box, Typography, styled } from '@mui/material';
+import { sequel85, sora } from 'src/fonts/fonts';
+
+export const NFTClaimingContainer = styled(Box)(({ theme }) => ({
+  backgroundColor: '#fdfbef',
```

Comment:
This seems to be already included in the ThemeProvider, using the deepMerge -> Can´t we use it from the theme then?